roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -97,15 +97,14 @@ policyConfig:
   bootstrapPolicyFile: {{ openshift_master_policy }}
   openshiftInfrastructureNamespace: openshift-infra
   openshiftSharedResourcesNamespace: openshift
-{# TODO: Allow users to override projectConfig items #}
 projectConfig:
-  defaultNodeSelector: "{{ openshift.master.default_node_selector | default("") }}"
-  projectRequestMessage: ""
-  projectRequestTemplate: ""
+  defaultNodeSelector: {{ openshift.master.default_node_selector }}
```

Comment:
@abutcher agreed, we should probably try to enforce quotes for string values, even though they aren't strictly necessary. It may help us avoid some parsing issues as well.